stackable-certs/src/ca/mod.rs

```diff
@@ -68,20 +68,58 @@ pub enum Error {
     ParseAuthorityKeyIdentifier { source: x509_cert::der::Error },
 }
 
+/// Defines all error variants which can occur when loading a CA from a
+/// Kubernetes [`Secret`].
+#[derive(Debug, Snafu)]
+pub enum SecretError<E>
```

Comment:
I would argue that this is implied by context (for now). Currently, this code is only used for Kubernetes specific tasks.

If this will ever change, we can rename it in the future.